enhancements/installer/component-selection.md

```diff
@@ -92,235 +97,185 @@ an install experience that disables components that aren't needed for my use cas
-from being part of the core(always installed) to being part of an optional capability.  In such cases,
+We will [consider][admission-webhook] adding an admission webhook to improve the UX for admins who attempt to remove an enabled capability, by replacing "notice `ImplicitlyEnabledCapabilities=True`" with "have your PATCH fail with a 400".
+
+###### Updates
+
+During updates, resources may move from being associated with one capability, to another, or move
+from being part of the core (always installed) to being part of an optional capability.  In such cases,
 the governing logic is that once a resource has been applied to a cluster, it must continue to be
-applied to the cluster(unless it is removed from the payload entirely, of course).  Furthermore,
+applied to the cluster (unless it is removed from the payload entirely, of course).  Furthermore,
 if any resource from a given capability is applied to a cluster, all other resources from that
 capability must be applied to the cluster, even if the cluster configuration would otherwise filter
 out that capability.  These are referred to as implicitly enabled capabilities and the purpose is to
 avoid scenarios like this that would break cluster functionality:
 
-```text
-1) initially registry resources are part of core(not defined in a capability)
-2) a cluster is installed, registry resources are applied to the cluster, admin chooses to 
+1. Initially, registry resources are part of core (not defined in a capability).
+2. A cluster is installed, registry resources are applied to the cluster, admin chooses to 
 disable optional capabilities by default.
-3) a new version of OCP moves the registry resources into an optional capability
-4a) during upgrade the registry resources are not applied to the cluster because the registry
-capability is not enabled
-OR
-4b) during upgrade, the existing registry resources continue to be applied(because we have a rule
-that says once a resource is applied, we will continue to reconcile it in future versions), but
-new registry resources added to the payload are not applied because they are filtered due to the
-registry capability not being enabled, resulting in critical registry resources not being applied
-and the registry potentially breaking.
-```
+3. A new version of OCP moves the registry resources into an optional capability.
+4. Either:
+    * During upgrade the registry resources are not applied to the cluster because the registry capability is not enabled, or
+    * During upgrade, the existing registry resources continue to be applied (because we have a rule that says once a resource is applied, we will continue to reconcile it in future versions).
+        But new registry resources added to the payload are not applied because they are filtered due to the registry capability not being enabled, resulting in critical registry resources not being applied and the registry potentially breaking.
```

Comment:
We should restructure or rewrite `But new registry resources added to the payload are not applied because they are filtered due to the registry capability not being enabled, resulting in critical registry resources not being applied and the registry potentially breaking.` as this is confusing with respect to what you want to communicate.

Reply:
```suggestion
       Either way, new registry-related resources are not going to be applied because the registry capability is not explicitly enabled.  This can result in critical registry resources not being applied, which could break the (already enabled/running) registry.
```

?

Reply:
This is pre-existing wording:

```console
$ git show 523c93d595 | grep -3 'new registry resources added to'
-OR
-4b) during upgrade, the existing registry resources continue to be applied(because we have a rule
-that says once a resource is applied, we will continue to reconcile it in future versions), but
-new registry resources added to the payload are not applied because they are filtered due to the
-registry capability not being enabled, resulting in critical registry resources not being applied
-and the registry potentially breaking.
-```
--
+4. Either:
+    * During upgrade the registry resources are not applied to the cluster because the registry capability is not enabled, or
+    * During upgrade, the existing registry resources continue to be applied (because we have a rule that says once a resource is applied, we will continue to reconcile it in future versions).
+        But new registry resources added to the payload are not applied because they are filtered due to the registry capability not being enabled, resulting in critical registry resources not being applied and the registry potentially breaking.
 
 This scenario is avoided by specifying that once a resource is applied to a cluster, it will
 always be reconciled in future versions, and that if any part of a capability is present,
```

Did you have a particular rephrasing you're looking for?

Reply:
I like what @bparees suggested

Reply:
I think there is some benefit to sketching how things can go wrong in both the "we stop managing previously-managed resources and let them drift in-cluster" and "we continue managing previously-managed resources, but no longer manage new-to-the-cap resources".  But if it's confusing to explain those cases, we've already picked a direction, and going into the specifics of why we think that was a good direction is no longer all that interesting.  So I've dropped this list in 523c93d5 -> 7907bdc8.